### PR TITLE
Daniel/cors

### PR DESCRIPTION
**Pull request by dantheli**
- enables CORS for our web app's potential URLs: (http://pollo.cornellappdev.com, https://pollo.cornellappdev.com)

### CODE REVIEW

**Comment by meganle** [comment]
How do I access the normal routes? I tried running it locally but all the routes give me an `Not allowed by CORS` error

File: src/API.js [hunk 1]
```diff
@@ -17,14 +18,18 @@ class API {
...
-        res.header('Access-Control-Allow-Headers',
-          'Origin, X-Requested-With, Content-Type, Accept');
-        next();
-      });
+
+    const whitelist = ['http://pollo.cornellappdev.com', 'https://pollo.cornellappdev.com'];
+    const corsOptions = {
+      origin: (origin, callback) => {
+        if (whitelist.indexOf(origin) !== -1) {
+          callback(null, true)
```
Comment (meganle) [inline src/API.js]: semicolons plz lol

File: src/API.js [hunk 2]
```diff
@@ -17,14 +18,18 @@ class API {
...
+        } else {
+          callback(new Error('Not allowed by CORS'))
```
Comment (meganle) [inline src/API.js]: here too

File: src/API.js (hunk 1 above)
Comment (dantheli) [reply src/API.js]: oops copy pasted

**Pull request merged**